flink-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/table/descriptors/KafkaValidator.java

```diff
@@ -92,21 +96,36 @@ private void validateStartupMode(DescriptorProperties properties) {
 		startupModeValidation.put(CONNECTOR_STARTUP_MODE_VALUE_GROUP_OFFSETS, noValidation());
 		startupModeValidation.put(CONNECTOR_STARTUP_MODE_VALUE_EARLIEST, noValidation());
 		startupModeValidation.put(CONNECTOR_STARTUP_MODE_VALUE_LATEST, noValidation());
-		startupModeValidation.put(
-			CONNECTOR_STARTUP_MODE_VALUE_SPECIFIC_OFFSETS,
-			key -> properties.validateFixedIndexedProperties(CONNECTOR_SPECIFIC_OFFSETS, false, specificOffsetValidators));
+
+		if (properties.containsKey(CONNECTOR_SPECIFIC_OFFSETS)) {
+			validateAndGetSpecificOffsetsStr(properties);
```

Comment:
Can you validate using validator style?

Reply:
the same reason above hostsStr validation